api/src/test/java/org/openmrs/api/UserServiceTest.java

```diff
@@ -1351,4 +1351,32 @@ public void changePasswordUsingSecretAnswer_shouldNotUpdatePasswordIfSecretIsNot
 		
 		userService.changePasswordUsingSecretAnswer("wrong answer", "userServiceTest2");
 	}
+	
+	/*
+	 * @see UserService#getUserByEmail(String)
+	 */
+	@Test
+	public void getUserByEmail_shouldGetUserByUsingEmail() {
+		executeDataSet(XML_FILENAME);
+		User user = userService.getUserByEmail("hank.williams@gmail.com");
+
+		assertNotNull("User with email hank.williams@gmail not found in database", user);
+	}
+	/*
+	 * @see UserService#getUserByActivationKey(String)
+	 */
+	@Test
+	public void getUserByActivationKey_shouldGetUserUsingActivationKey() {
+		executeDataSet(XML_FILENAME);
+		
+		User user = userService.getUserByActivationKey("XbklsieiskeuztecEnPf");
+		
+		assertNotNull("User with activationKey ", user); 
+		if(user!=null) {
```

Comment:
This is bad practice in a unit test, you should be asserting that it's not null instead